console/src/main/scripts/plugins/accounts/ts/persona.ts

```diff
@@ -49,12 +49,42 @@ module HawkularAccounts {
 
     public loadData():void {
       this.loading = true;
-      this.currentPersona = this.HawkularAccount.Persona.get({id: 'current'},
+      let personaToGet = 'current';
+
+      let lastPersonaId = localStorage.getItem('lastPersona');
```

Comment:
Aha localStorage. Good to know its being used. QE might also want to be aware of this.